distribution/sender/routes/database-event-reader.xml

```diff
@@ -0,0 +1,115 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+
+    <route id="db-event-reader">
+        <from uri="scheduler:retry?initialDelay=1000&amp;delay=10000" />
```

Comment:
Don't set scheduler to retry, use something like reader

Reply:
Ok